src/components/box/box.test.tsx

```diff
@@ -76,6 +76,40 @@ it("applies the boxShadow styling correctly when a design token is passed in", (
   expect(box).toHaveStyle(`box-shadow: var(--boxShadow100)`);
 });
 
+it("applies the correct styling from the cssProps", () => {
+  render(
+    <Box
+      width="100px"
+      height="100px"
+      opacity="25%"
+      color="yellow"
+      data-role="box"
+    />,
+  );
+
+  const box = screen.getByTestId("box");
+  expect(box).toHaveStyle(`width: 100px`);
```

Comment:
**suggestion(non-blocking):** `toHaveStyle` accepts an object with multiple values, so you could pass these in as one assertion. I'm OK with the way you've done it, though. Feel free to ignore this comment if you prefer your way 😄